src/controllers/member.js

```diff
@@ -71,7 +71,32 @@ const updateMember = (req, res) => {
     .catch((error) => res.status(500).json(error));
 };
 
+const getAllMembers = (req, res) => {
+  Member.find()
+    .then((members) => res.status(200).json({
+      message: 'Complete members list',
+      data: members,
+      error: false,
+    }))
+    .catch((error) => res.status(500).json({
+      message: 'An error ocurred',
+      error,
+    }));
+};
+
+const getById = (req, res) => {
+  const { id } = req.params;
+  Member.findById(id)
+    .then((member) => res.status(200).json({
```

Comment:
- Inside .then you should add an if else for when the user is not found
- .catch missing (status 500)